app-web/plugins/gatsby-source-github-all/utils/plugins.js

```diff
@@ -164,9 +177,26 @@ const markdownResourceTypePlugin = (extension, file) => {
   return file;
 };
 
+/**
+ * unfurls from an HTTP request and appends .unfurl metadata property
+ * @param {String} extension 
+ * @param {String} file 
+ * @returns the modified file
+ */
+const externalLinkUnfurlPlugin = async (extension, file) => {
+  // does file have a resource path and is it a valid url?
+  if (file.metadata.resourcePath && validUrl.isUri(file.metadata.resourcePath)) {
+    const { body: html, url } = await got(file.resourcePath);
+    const metadata = await metascraper({ html, url });
+    file.metadata.unfurl = createUnfurlObj('external', metadata);
```

Comment:
Keywords like 'external' might go well in `constants.js`. That way if you need to change it later you can just edit one place rather than search-replace in the code base. Only change this if it makes sense (makes the code more maintainable).

```js
export const LINK_TYPE = {
  EXTERNAL: 'external',
  INTERNAL: 'internal',
};```

Reply:
good point, i'll make that change !